nexus/examples/config-second.toml

```diff
@@ -142,6 +142,7 @@ region_snapshot_replacement_garbage_collection.period_secs = 30
 region_snapshot_replacement_step.period_secs = 30
 region_snapshot_replacement_finish.period_secs = 30
 tuf_artifact_replication.period_secs = 300
+tuf_artifact_replication.min_sled_replication = 3
```

Comment:
I think this one should probably be 1-2 but this is really unimportant -- it's only used for people running the stack with `cargo run` with simulated sled agents.